docs/workshops/overview.md

```diff
@@ -6,4 +6,5 @@ See below for lesson materials for workshops that have been held at Arcadia.
 
 | Date | Tutorial | Length | Description | Recording |
 |------|----------|--------|-------------|-----------|
+| Oct 6, 2023 | [Git & GitHub for Hacktoberfest](20231006-github-hacktoberfest/lesson.md) | optional | This lesson is an optional refresher for participants in our Hacktoberfest mini-hackathon. It covers how to make contributions to repositories that you don't own via forks and pull requests. | |
```

Comment:
I know it is not related to this PR, but can you also update the schedule on the other aug page?

Reply:
ya i'll do it either EOD today or tomorrow!